custom_components/garmin_connect/__init__.py

```
@@ -421,8 +432,7 @@ async def _async_update_data(self) -> dict:
                 "Too many request error occurred during update: %s", err)
             return {}
         except GarminConnectConnectionError as err:
-            _LOGGER.error(
-                "Connection error occurred during update: %s", err)
+            _LOGGER.error("Connection error occurred during update: %s", err)
```

Ruff (0.14.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)